### ISSUE TITLE
Tensorflow 1.15 Installation Problem in python 3.8+ version

### ISSUE THREAD

**Issue by Dhyani01** [saroz014/Plant-Diseases-Recognition#25]
If your python version is 3.7 or less this requirement.txt installation will work great, 
Those who have python version 3.8+ will face some issues while installing Keras 1.15

For Python version 3.8+ first, install TensorFlow by
>pip install tensorFlow

it will install TensorFlow version 2.5
Do these following steps in order to make the project working :

1. Replace this line graph = tf.get_default_graph() by graph= tf.compat.v1.get_default_graph() in deeplearning.py
2. Remove this import,
>from keras.models import load_model

 by this.

>from tensorflow import keras

3. Model is loaded in deeplearning.py instead load it inside the working code 

>with graph.as_default():
>    model = tf.keras.models.load_model('AlexNetModel.hdf5')
>   prediction = model.predict(image_array)

4: The code is using Keras library for importing images you can now import it by Image library.

comment this line# img = image.load_img('sample_image.png', target_size=(224, 224))
comment this line# img = image.img_to_array(img)
comment this line# img = np.expand_dims(img, axis=0)
comment this line# img = img/255
comment this line# print(type(img))
comment this line# img = tf.keras.preprocessing.image.array_to_img(img_data)

Now replace those lines by these lines : 
> input_image=Image.open("sample_image.png")
> input_image = input_image.resize((224,224), resample=Image.LANCZOS)
> image_array=np.array(input_image)
> image_array=image_array.astype('float32')
> image_array/=255
> image_array=image_array.reshape(1,224,224,3)

These steps will fix your code according to the version.

**Comment by garudaonekh** [comment]
Hello,
Can you provide more detail on step4? I got stuck when I upload image, it errors with Image library.

**Comment by Dhyani01** [comment]
@garudaonekh  If you have done all the steps from 1 to 3 correctly. Then for step 4 go to view.py inside plant_app folder and replace those lines above with the new ones.

**Comment by ramyaperumandla24** [comment]
@Dhyani01  @saroz014 @garudaonekh 
![Screenshot (17)](https://user-images.githubusercontent.com/57004716/167792809-a346e699-b1b2-4b4b-9be2-4207a6e50365.png)

I am facing this issue after clicking on predict button, can u please help with this?
